uber/templates/registration/index_base.html

```diff
@@ -11,11 +11,29 @@
 </script>
     {% include "check_in.html" %}
 
+  {% if c.REG_USES_BARCODES %}
+    {% set detect_barcode_scan = True %}
+    <script type="text/javascript">
+        var setBarcodeField = function () {
+            if ($('input[name=badge_num]').length === 1) {
+                $barcodeField = $('input[name=badge_num]');
```

Comment:
This is a global variable? Which makes it accessible from anywhere on the page?

Reply:
It, uh... aspires to be a global variable?

I'll double-check this.